src/yz_extractor.erl

```diff
@@ -32,6 +32,7 @@
                       {"application/json",yz_json_extractor},
                       {"application/riak_counter", yz_dt_extractor},
                       {"application/riak_hll", yz_dt_extractor},
+                      {"application/riak_gset", yz_dt_extractor},
```

Comment:
@russelldb this need to be alphabetically ordered. @lukebakken added a test to ensure they were. If they aren't properly ordered the extraction won't work.